traptor/traptor.py

```diff
@@ -935,9 +955,10 @@ def run(self):
             self.twitter_rules = self._make_twitter_rules(self.redis_rules)
             self.logger.debug("Twitter rules: {}".format(self.twitter_rules.encode('utf-8')))
 
-            # Make the rule counters
+            # Make the rule and limit message counters
             if self.traptor_type != 'locations':
                 self._make_rule_counters()
+                self._make_limit_message_counter()
```

Comment:
We receive a lot of limit messages from location Traptors. However, this information is being used for rule rebalancing, which only applies to follow and track traptors. I will add the existence check for the counter to ensure we don't run into the `NoneType` issue you mention. Thanks!